GridKit/Model/PhasorDynamics/SynchronousMachine/GENROUwS/GenrouImpl.hpp

```diff
@@ -303,6 +306,38 @@ namespace GridKit
       return 0;
     }
 
+    /**
+     * @brief verify method checks that attached signals are also linked
+     */
+    template <class ScalarT, typename IdxT>
+    int Genrou<ScalarT, IdxT>::verify() const
+    {
+      static constexpr auto PM  = GenrouExternalVariables::PM;
+      static constexpr auto EFD = GenrouExternalVariables::EFD;
+
+      int ret = 0;
+
+      if (signals_.template isAttached<PM>())
+      {
+        if (!signals_.template isLinked<PM>())
+        {
+          Log::error() << "Genrou: pmech signal attached with no linked governor\n";
+          ret += 1;
```

Comment:
i'm dubious of the value of incrementing this return value type. i saw this pattern in resolve too and i saw it cause more problems than it solved. i think just returning "1" on error or having meaningfully defined error codes is more useful.